source/configure/calls-deployment.rst

```diff
@@ -455,4 +446,4 @@ On the server side, run the following:
 
    sudo tcpdump -n port 8443
 
-This command will output information (i.e. source and destination addresses) for all the network packets being sent or received through port ``8443``. This is a good way to check whether data is getting in and out of the instance and can be used to quickly identify network configuration issues.
\ No newline at end of file
+This command will output information (i.e. source and destination addresses) for all the network packets being sent or received through port ``8443``. This is a good way to check whether data is getting in and out of the instance and can be used to quickly identify network configuration issues.
```

Comment:
Hmm, why is this line marked as modified?

Reply:
Maybe a new line or white space, not sure.

Reply:
🤷